src/inventories/batches/model/BatchCreationAdjustment.ts

```diff
@@ -0,0 +1,9 @@
+export default interface BatchCreationAdjustment {
+  sku: string;
+  quantity: number;
+  idempotentKey?: string;
```

Comment:
Idempotent key is not optional. Please make this a required field in this object.